deployment/quick-start/services-configuration.yaml.template

```diff
@@ -74,16 +74,10 @@ frameworklauncher:
 
 
 
-restserver:
-  # port for rest api server
-  server-port: 9186
-  # secret for signing authentication tokens, e.g., "Hello PAI!"
+rest-server:
   jwt-secret: pai-secret
```

Comment:
Maybe in the template, we could only left the necessary configuration. And remove the configuration with the default value.  Because I found that some of the key-value are same with the default.

Reply:
Sorry for misunderstood both files, now inverted them.